### PR TITLE
MAUI-4126-Included changes in image name caption for DataForm UG

### PR DESCRIPTION
**Pull request by devanathan-vaithiyanathan**
I have update image name caption for the following DataForm UG Content:
* Data editors
* Dataform editor settings

### CODE REVIEW

**Comment by karthikraja-arumugam** [comment]
rebuild

File: MAUI/DataForm/DataForm-settings.md [hunk 1]
```diff
@@ -283,7 +283,7 @@ private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs
 {% endhighlight %}
 {% endtabs %}
 
-![PlaceHolderText](images/editor-settings/placeholder-text.png)
+![Placeholder text for .NET MAUI DataForm.](images/editor-settings/placeholder-text.png)
```
Comment (karthikraja-arumugam) [inline MAUI/DataForm/DataForm-settings.md:286]: Editor placeholder text in .NET MAUI DataForm.

File: MAUI/DataForm/DataForm-settings.md [hunk 2]
```diff
@@ -310,7 +310,7 @@ private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs
 {% endhighlight %}
 {% endtabs %}
 
-![PlaceHolderTextColor](images/editor-settings/placeholder-text-color.png)
+![Placeholder text color for .NET MAUI DataForm.](images/editor-settings/placeholder-text-color.png)
```
Comment (karthikraja-arumugam) [inline MAUI/DataForm/DataForm-settings.md:313]: Editor placeholder text color for .NET MAUI DataForm.

File: MAUI/DataForm/DataForm-settings.md [hunk 3]
```diff
@@ -338,7 +338,7 @@ private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs
 {% endhighlight %}
 {% endtabs %}
 
-![Padding](images/editor-settings/changing-padding.png)
+![Padding for .NET MAUI DataForm.](images/editor-settings/changing-padding.png)
```
Comment (karthikraja-arumugam) [inline MAUI/DataForm/DataForm-settings.md:341]: Editor padding in .NET MAUI DataForm.

File: MAUI/DataForm/DataForm-settings.md [hunk 4]
```diff
@@ -477,7 +477,7 @@ private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs
 {% endhighlight %}
 {% endtabs %}
 
-![HideLabelName](images/editor-settings/hide-label-name.png)
+![Hide label name for .NET MAUI DataForm.](images/editor-settings/hide-label-name.png)
```
Comment (karthikraja-arumugam) [inline MAUI/DataForm/DataForm-settings.md:480]: Hide label in .NET MAUI DataForm.

File: MAUI/DataForm/DataForm-settings.md [hunk 5]
```diff
@@ -541,7 +541,7 @@ private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs
 {% endhighlight %}
 {% endtabs %}
 
-![leading-label-icon](images/layout/leading-label-icon.png)
+![Leading label icon for .NET MAUI DataForm.](images/layout/leading-label-icon.png)
```
Comment (karthikraja-arumugam) [inline MAUI/DataForm/DataForm-settings.md:544]: Leading label icon in .NET MAUI DataForm.

File: MAUI/DataForm/DataForm-settings.md [hunk 6]
```diff
@@ -605,7 +605,7 @@ private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs
 {% endhighlight %}
 {% endtabs %}
 
-![LabelTextStyle](images/editor-settings/label-text-style.png)
+![Label text style for .NET MAUI DataForm.](images/editor-settings/label-text-style.png)
```
Comment (karthikraja-arumugam) [inline MAUI/DataForm/DataForm-settings.md:608]: Label text style in .NET MAUI DataForm.

File: MAUI/DataForm/DataForm-settings.md [hunk 7]
```diff
@@ -630,7 +630,7 @@ private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs
 {% endhighlight %}
 {% endtabs %}
 
-![EditotTextStyle](images/editor-settings/editor-text-style.png)
+![Editor text style for .NET MAUI DataForm.](images/editor-settings/editor-text-style.png)
```
Comment (karthikraja-arumugam) [inline MAUI/DataForm/DataForm-settings.md:633]: Editor text style in .NET MAUI DataForm.

File: MAUI/DataForm/Editors.md [hunk 8]
```diff
@@ -216,7 +216,7 @@ public string Name { get; set; }
 {% endhighlight %}
 {% endtabs %}
 
-![DataForm-Text-Editor](images/editors/dataform-text-editor.png)
+![Text editor for .NET MAUI DataForm.](images/editors/dataform-text-editor.png)
```
Comment (karthikraja-arumugam) [inline MAUI/DataForm/Editors.md:219]: Text editor in .NET MAUI DataForm.

File: MAUI/DataForm/Editors.md [hunk 9]
```diff
@@ -273,7 +273,7 @@ public string Address { get; set; }
 {% endhighlight %}
 {% endtabs %}
 
-![MultilineTextEditor](images/editors/dataform-multiline-text-editor.png)
+![Multiline Text editor for .NET MAUI DataForm.](images/editors/dataform-multiline-text-editor.png)
```
Comment (karthikraja-arumugam) [inline MAUI/DataForm/Editors.md:276]: Multiline Text editor in .NET MAUI DataForm.

Update all editors caption like this

File: MAUI/DataForm/DataForm-settings.md (hunk 2 above)
Comment (devanathan-vaithiyanathan) [reply MAUI/DataForm/DataForm-settings.md:313]: changed

File: MAUI/DataForm/DataForm-settings.md (hunk 3 above)
Comment (devanathan-vaithiyanathan) [reply MAUI/DataForm/DataForm-settings.md:341]: changed

File: MAUI/DataForm/Editors.md (hunk 8 above)
Comment (devanathan-vaithiyanathan) [reply MAUI/DataForm/Editors.md:219]: changed

File: MAUI/DataForm/Editors.md (hunk 9 above)
Comment (devanathan-vaithiyanathan) [reply MAUI/DataForm/Editors.md:276]: updated

File: MAUI/DataForm/DataForm-settings.md (hunk 7 above)
Comment (devanathan-vaithiyanathan) [reply MAUI/DataForm/DataForm-settings.md:633]: updated

**Review by Jeyasri-Murugan — APPROVED** [review]

**Pull request merged**